app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas/_after-accordion.md

```diff
@@ -29,10 +29,10 @@ learn more.
 
 ### Contact
 
-Sign up for an [online Q&A for returners](/events).
+To find out more about returning to England to teach, you can:
 
-Email us at teach.inengland@education.gov.uk.
-
-Join us on [Facebook](https://www.facebook.com/getintoteaching),
-[Instagram](https://www.instagram.com/get_into_teaching/) and
-[Twitter](https://twitter.com/getintoteaching).
+* sign up for an [online Q&A for returners](/events)
```

Comment:
@gemmadallmandfe not to overcomplicate this PR but ... should we be linking to the generic events page if the link text is 'online Q&A for returners'?

Reply:
this is a tricky one as we don't have a category for returners events so there is nothing to link them to!

we could link to pre-filtered results for online Q&As if you think that is better

Reply:
probs link to online Q&As rather than just the events page and remove 'returners' from the link text so it's more accurate?